### DISCUSSION TITLE
Dimana saya bisa melihat layanan yang menggunakan aws akun free tier dan durasinya?

### DISCUSSION THREAD

**Discussion by misskecupbung** [awsugid/forum#2] [Q&A]
Selamat siang, permisi saya member baru~
Izin mau bertanya, dimana lihat layanan apa saja yg di pakai dan durasinya dalam aws akun free tier? Terimakasih

**Comment by misskecupbung** [comment]
Buat lihat pemakaian free tier bisa:

1. Melalui AWS Management Console
2. Di menu Billing > Free Tier
3. Untuk melihat resource apa saja yang digunakan bisa menggunakan AWS Resource Explorer

**Comment by willysr** [comment]
https://aws.amazon.com/free/
